roles/base/tasks/main.yml

```diff
@@ -54,16 +76,24 @@
   copy:
     dest: /etc/profile.d/00golang.sh
     content: "export PATH=/opt/golang/bin:/usr/local/go/bin:$PATH\nexport GOPATH=/opt/golang"
+- name: remove older docker
```

Comment:
This is minor but I think instead of a task to remove docker, we should enhance/fix the task to install docker to do the right thing i.e. only install/upgrade docker if we are at version lower that 1.9? This would help us have all docker version specific logic in one task.